docs/basics.md

```diff
@@ -156,15 +187,46 @@ A running Connaisseur instance can be updated by a Helm upgrade of the current r
     helm upgrade connaisseur connaisseur/connaisseur -n connaisseur --wait -f values.yaml
     ```
 
+=== "Kubernetes manifests"
 
+    Adjust your local Kubernetes manifests (e.g. `deploy.yaml`) as required and upgrade via delete and reinstall:
+
+    ```bash
+    kubectl delete -f deploy.yaml -n connaisseur
+    kubectl apply -f deploy.yaml -l 'app.kubernetes.io/component=core' -n connaisseur
+    ```
+    !!! note
+        Rolling upgrades as with Helm might also be possible, but might require further configuration.
+        Insights are welcome :pray:
 
 ### Delete
 
 Just like for installation, Helm can also be used to delete Connaisseur from your cluster:
 
-```bash
-helm uninstall connaisseur -n connaisseur
-```
+
+=== "Git repo"
+
+    Uninstall via Helm:
+
+    ```bash
+    helm uninstall connaisseur -n connaisseur
```

Comment:
Would this be a place where we use the simpler versions of `make uninstall`? (Would then need the same during install)

Reply:
thought about it, would be nice, but needs makefile support 🤷